api/BUILD

```diff
@@ -119,25 +89,73 @@ proto_library(
         "//envoy/config/resource_monitor/fixed_heap/v2alpha:pkg",
         "//envoy/config/resource_monitor/injected_resource/v2alpha:pkg",
         "//envoy/config/retry/previous_priorities:pkg",
+        "//envoy/config/route/v3alpha:pkg",
         "//envoy/config/trace/v2:pkg",
         "//envoy/config/trace/v2alpha:pkg",
         "//envoy/config/trace/v3alpha:pkg",
         "//envoy/config/transport_socket/alts/v2alpha:pkg",
         "//envoy/config/transport_socket/tap/v2alpha:pkg",
-        "//envoy/config/transport_socket/tap/v3alpha:pkg",
         "//envoy/data/accesslog/v2:pkg",
         "//envoy/data/accesslog/v3alpha:pkg",
         "//envoy/data/cluster/v2alpha:pkg",
         "//envoy/data/core/v2alpha:pkg",
         "//envoy/data/core/v3alpha:pkg",
         "//envoy/data/tap/v2alpha:pkg",
         "//envoy/data/tap/v3alpha:pkg",
+        "//envoy/extensions/access_loggers/grpc/v3alpha:pkg",
```

Comment:
One useful thing to verify is that the number of extensions packages in the new hierarchy matches those in the old one. Some extensions won't have bumped to v3alpha (e.g. if they are very simple or only rely on raw proto types).

Reply:
All moved package are marked need to upgrade in type_whisperer so if we do the move they will be upgraded.

Reply:
Sure, make sense. It would be a good belt-and-braces to verify the count if you have a sec and haven't already done so.